### DISCUSSION TITLE
[#991906] Input field bugs

### DISCUSSION THREAD

**Discussion by tryashtar** [Mojang/minecraft-editor#32] [Bugs]
1. Can't use tab to navigate fields
2. Selection box exists (e.g. ctrl+A) but is invisible
3. Can't use ctrl+z to undo edits to fields
4. Blinking caret is so thin to be almost invisible
5. Clicking with mouse to move caret does not work

**Comment by minecraftchloe** [comment]
Updated with internal ID for tracking!
